### DISCUSSION TITLE
Tips wanted: Creating website Strapi CMS and handing over to clients?

### DISCUSSION THREAD

**Discussion by Devalo** [strapi/strapi#8075] [Legacy Topics]
Hi guys,
Apart from developing web software, I'm also creating a bunch of websites for clients.
I'm allergic to php and WordPress (even though I have supricingly little experience with it lol).

I just rebuilt my company site in Next.js and Strapi. 
The Next.js app is built as a static website (SSG), but will fetch content from Strapi at a given rate.
This resulted in an extremely fast website (as its html), even though it's backed by an external API (Strapi).

Ususally what I would have to do on a client website, is to create a admin area in which they could update
their website content. With Strapi you kind of get this functionallity for free.

I was thinking about using this solution for client websites. 
Is anyone else using Strapi for customer websites? 
What modifications are you doing to the Strapi dashboard, to make it look less intimidating to a tech savy client?

I will, of course try to sell a maintenance plan, so that I can provide service to the websites my self, but sometimes
the client wants to do all that themselves. I want to provide them the best possible solution. That aint WordPress..

Any tips or experiences on this subject? :) 

Regards
Stephan Valois

**Comment by derrickmehaffy** [comment]
As we are planning to decommission GitHub discussions in early 2021 I was hoping I could ask you to move your question over to our forums, specifically in the Q&A Section. My hope is that others with a similar issue will be able to search and find the information they are looking for much easier. :smiley: 

Here is a link to the Q&A Section: https://forum.strapi.io/c/support

**Comment by Devalo** [comment]
Didn't even know there was a forum! Of course :-)
